react/tic-tac-toe/src/app/index.js

```diff
@@ -1,34 +1,41 @@
 import React from 'react';
-import { BrowserRouter as Router, Route, Switch, Redirect } from 'react-router-dom';
+import { BrowserRouter as Router, Route, Redirect } from 'react-router-dom';
 import { connect } from 'react-redux';
 import PropTypes from 'prop-types';
 
 import Game from './screens/Game';
 import Login from './screens/Login';
+import Profile from './screens/Profile';
+import Topbar from './screens/Topbar';
 
 import '../scss/application.scss';
 
 function App(props) {
-  const { token } = props;
+  const { token, selectedUser } = props;
   return (
     <Router>
-      <Switch>
+      <div>
```

Comment:
why did you take out the Switch? i think taking the TopBar route out of the switch is enough if you just want to show the topBar everywhere